### PR TITLE
Ptask01 1

### PR DESCRIPTION
**Pull request by pskrzybor**

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -11,5 +11,57 @@
...
+			<version>5.4.21.Final</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/ch.qos.logback/logback-classic -->
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
+			<version>1.2.3</version>
+			<scope>test</scope>
+		</dependency>
+
```
Comment (juliabaar) [inline backend/pom.xml:60]: please add mysql dependency

File: pom.xml [hunk 2]
```diff
@@ -49,17 +49,31 @@
...
 			</dependency>
+
+			<!-- https://mvnrepository.com/artifact/mysql/mysql-connector-java -->
+
+			<dependency>
+				<groupId>mysql</groupId>
+				<artifactId>mysql-connector-java</artifactId>
+				<version>8.0.21</version>
+			</dependency>
+
```
Comment (juliabaar) [inline pom.xml:60]: Please add logback dependency

File: backend/pom.xml [hunk 3]
```diff
@@ -11,5 +11,57 @@
...
+			<artifactId>slf4j-api</artifactId>
+			<version>1.7.30</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
+			<version>1.2.3</version>
+		</dependency>
```
Comment (aksk08) [inline backend/pom.xml:45]: This is dangerous - duplication

File: backend/pom.xml [hunk 4]
```diff
@@ -11,5 +11,57 @@
...
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-core</artifactId>
+			<version>5.4.21.Final</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/ch.qos.logback/logback-classic -->
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
+			<version>1.2.3</version>
+			<scope>test</scope>
```
Comment (aksk08) [inline backend/pom.xml:58]: THird time :)

**Pull request closed without merging**